src/WrappedMToken.sol

```diff
@@ -39,17 +39,17 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
 
     /* ============ Variables ============ */
 
-    /// @dev Registrar key holding value of whether the earners list can be ignored or not.
-    bytes32 internal constant _EARNERS_LIST_IGNORED = "earners_list_ignored";
+    /// @inheritdoc IWrappedMToken
+    bytes32 public constant EARNERS_LIST_IGNORED_KEY = "earners_list_ignored";
```

Comment:
nitpicking:

IS_EARNERS_LIST_IGNORED
CLAIM_RECIPIENT_PREFIX 

look more concise

Reply:
The key already established in M as "earners_list_ignored", and it is a key, so `EARNERS_LIST_IGNORED_KEY` is apt. `IS_EARNERS_LIST_IGNORED` sounds like a `bool` in comparison.

In the current iteration, the claim recipient is defaulted to the account itself, and can be overridden by the Registrar, so I feel claim `LAIM_OVERRIDE_RECIPIENT` is apt, plus, it is a key prefix, so  `CLAIM_OVERRIDE_RECIPIENT_KEY_PREFIX`.